RIOT-OS/RIOT
# Small fatfs usability fixes

### Contribution description

This is a collection of the small things that make life easier for someone who wants to give fatfs a try:

* Name the tarball of tests/pkg_fatfs .tar.bz2 (tools like [atool](https://www.nongnu.org/atool/) do look at extensions)
* Make the extraction routine work even when bin/ has not yet been created (with Makefile style fixes as bycatch)
* **Tell user what to do when enabling fatfs to get a usable example (because the disk can't format itself)**

### Testing procedure

* In `tests/pkg_fatfs`, run (there is not automation for this, is there?):

    ```
    $ make image all term
    > mount 0
    mounting file system image...
    [OK]
    Volume name: 
    0,124 GiB of 0,124 GiB available
    > ls
    //TEST.TXT
    ```

* In `examples/filesystem`, follow the new set of instructions and observe that indeed the file system becomes usable with a TEST.TXT present from startup.
